src/platform/android/DeviceInstanceInfoProviderImpl.cpp

```diff
@@ -73,6 +73,32 @@ CHIP_ERROR DeviceInstanceInfoProviderImpl::GetProductLabel(char * buf, size_t bu
     return Internal::AndroidConfig::ReadConfigValueStr(Internal::AndroidConfig::kConfigKey_ProductLabel, buf, bufSize, dateLen);
 }
 
+CHIP_ERROR DeviceInstanceInfoProviderImpl::GetDeviceName(MutableCharSpan & deviceNameSpan)
+{
+    CHIP_ERROR err;
+    size_t deviceNameSize = 0;
+    char androidDeviceName[ConfigurationManager::kMaxDeviceNameLen];
+    MutableCharSpan androidDeviceNameSpan(androidDeviceName);
+
+    err = Internal::AndroidConfig::ReadConfigValueStr(Internal::AndroidConfig::kConfigKey_MfrDeviceName,
+                                                      androidDeviceNameSpan.data(), androidDeviceNameSpan.size(), deviceNameSize);
+
+    if (err = CHIP_NO_ERROR)
```

Comment:
I believe this needs to say err == CHIP\_NO\_ERROR? (Not the single =)

Reply:
fixed.